app/views/admin/invoices/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Admin Invoices Index</h1>
+<ul>
+<% @invoices.each do |invoice| %>
+<li> <%= link_to "#{invoice.id}", admin_invoice_path(invoice) %> </li>
```

Comment:
I like that you used the new _path syntax. I'm going to update my old ones when I have the chance and try to get used to it.